### ISSUE TITLE
expand on 'Evil bit research idea'

### ISSUE THREAD

**Issue by DinisCruz** [DinisCruz/Book_SecDevOps_Risk_Workflow#23]
http://blog.diniscruz.com/2013/12/trying-to-add-evil-bit-to.html (find more up-to-date content)

As a great example of the kind of 'advanced research' that should be done by AppSec teams + Security Champions (across multiple companies)

**Comment by davevs** [comment]
Not sure, but I think that is what xanitizer does: https://www.rigs-it.net/index.php/product.html

In general I think static analysis is the future, especially in a combination of code, byte code and dynamic memory code analysis.

**Comment by DinisCruz** [comment]
Yeah I'm a big fan of Static Analysis (tons of SAST technology in the O2 Platform)

xanitizer looks good, have you used it?

This license is weird https://www.rigs-it.net/index.php/product/license.html , it would be great it is was open source (and on a GitHub repo)

**Comment by davevs** [comment]
Not yet, but one of the scrum teams at my client is going to testdrive it this week. I will share my evaluation.

**Comment by davevs** [comment]
First results on xanitizer look good. They use findbugs+securityrules, OWASP dependency checker, and their own taint analysis. There are some minor issues with recognizing entry points, but you can add them manually. Results make sense and no obvious false positives were identified. Interface on standalone version is clean and logical. Next step is testing Jenkins headless mode and checking behaviour on next iteration.
